docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,255 @@
+sample application.
+
+Before starting, we invit you to refer to the [Prerequisites](#prerequisites)
+section to make sure you have all the needed components configured on your K8s
+cluster.
+
+The quick start will then consist into three main steps:
+1. [Create a PostgreSQL database instance](#creating-a-database-instance)
+2. [Deploy the application](#deploying-an-application)
+3. [Connect the application to the database with Service Binding](#connecting-the-application-to-a-backing-service)
+
+## Prerequisites
+
+In order to follow the quick start, you'll need the following tools installed and configured:
+
+- Kubernetes cluster (**Note:** You can use [minikube](https://minikube.sigs.k8s.io/) or
+  [kind](https://kind.sigs.k8s.io/), locally)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Service Binding Operator](installing-service-binding)
+
+# Create a PostgreSQL database instance
+
+The application is going to use to a PostgreSQL database backend which can be
+setup using the [Crunchy PostgreSQL operator from OperatorHub.io][crunchy].
+
+The installation of the operator doesn't create a database instance itself, so
+we need to create one.
+
+1. To create a database instance, you need to create custom resource `Pgcluster`
+   and that will trigger the operator reconciliation.  For convenience, run this
+   command to create `Pgcluster` custom resource:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/pgcluster.yaml
+```
+
+To enable binding, the `Pgcluster` resource has these annotions:
+
+```
+service.binding/database: path={.spec.name}
+service.binding/host: path={.spec.name}
+service.binding/port: path={.spec.port}
+```
+
+The annotions points to the values of `database`, `host`, and `post` in resource
+attributes.  For more details, refer [Exposing binding
+data](../exposing-binding-data/intro-expose-binding) section.
+
+2. Once the database is created, we need to ensure all the pods in
+   `my-postgresql` namespace are running (it will take few minutes):
+
+```bash
+kubectl get pod -n my-postgresql
+```
+
+You should see output something like this:
+
+```
+NAME                                          READY   STATUS    RESTARTS   AGE
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+The database has been created and is empty at this stage. We now need to set its
+schema and we will also inject a sample data set, so we can play around with the
+application.
+
+3. You can initialize the database with the schema and sample data using this
+   command:
+
+```bash
+bash <(curl -s https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/init-database.sh)>
```

Comment:
can you move the yaml under that documentation repo and create a dedicated "resources" folder for that?

Reply:
Done. 0e153c07f9d36cd93a8bd8121c173de04825a7d0